db/migrate/20221118102259_create_positions.rb

```diff
@@ -1,10 +1,11 @@
 class CreatePositions < ActiveRecord::Migration[7.0]
   def change
     create_table :positions do |t|
-      t.string :name
-      t.text :description
-      t.datetime :deadline
-      t.integer :available
+      t.string :name, null: false
+      t.text :description, null: false
+      t.datetime :date_starts_at, null: false
+      t.datetime :date_ends_at, null: false
```

Comment:
name can still be improved started_at, ends_at, vacancies

Reply:
done